### ISSUE TITLE
stern not found if install stern by kubectl krew

### ISSUE THREAD

**Issue by karnzx** [andrea-falco/lens-multi-pod-logs#3]
Hi, First of all I think this couldn't really be an issue at all but it will be greatly appreciated if the plugin cover this little problem.

I install stern with krew by `kubectl krew install stern` in my powershell teminal.
To use `stern` I have to run `kubectl stern` instead of single `stern` command
so when using this plugin on lens I got this error.
```powershell
PS C:\Users\user> stern deployment/web --color auto --namespace project_namespace --since 1s
stern : The term 'stern' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or 
if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ stern deployment/web --color auto --namespace project_n ...
+ ~~~~~
    + CategoryInfo          : ObjectNotFound: (stern:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

My solution is to mapping `kubectl stern` command to `stern` for example in powershell terminal. 
```powershell
function stern {
    kubectl stern $args
}
```

**Comment by andrea-falco** [comment]
Hi @karnzx, during next days I'll try to do something to fix this "issue".

I don't know if Lens Extension API have something to do this, but in any case I could create a specific settings (in Lens/OpenLens settings page) to enable in this case.

**Comment by andrea-falco** [comment]
Hi @karnzx, I have just released a new version (v0.3.3) which includes a new preference to flag if you have installed stern via krew.

You can see below a screeenshot of the new preference:

![lens-multi-pod-logs_krew-preference](https://github.com/andrea-falco/lens-multi-pod-logs/assets/23004377/058fc439-acc1-4149-a0e2-953ab4678f1e)

**Comment by karnzx** [comment]
wonderful, thank you!
